### PR TITLE
Simplify log level environment variable usage

### PR DESCRIPTION
**Pull request by sterchelen**
- Add _LEVEL as suffix to CONSUL_TEMPLATE_LOG
- Remove CT_LOG
- Add reference to the environment variable on the readme file

ref #1328

### CODE REVIEW

**Comment by sterchelen** [comment]
Hey, any news about this PR?

File: config/config.go [hunk 1]
```diff
@@ -525,8 +525,7 @@ func (c *Config) Finalize() {
 
 	if c.LogLevel == nil {
 		c.LogLevel = stringFromEnv([]string{
-			"CT_LOG",
-			"CONSUL_TEMPLATE_LOG",
+			"CONSUL_TEMPLATE_LOG_LEVEL",
```
Comment (eikenb) [inline config/config.go:528]: How about we just make it all 3. I really don't like breaking backwards compatibility unless absolutely necessary, but I think the new one is better. Thanks.

Comment (sterchelen) [reply config/config.go:528]: Yes agree with you. I will do it this week.

Comment (sterchelen) [reply config/config.go:520]: @eikenb As expected, just adding the new env variable

**Pull request merged**